activerecord/test/cases/adapters/postgresql/geometric_test.rb

```diff
@@ -167,16 +167,19 @@ def test_legacy_mutation
 end
 
 class PostgresqlGeometricTest < ActiveRecord::PostgreSQLTestCase
+  include SchemaDumpingHelper
+
   class PostgresqlGeometric < ActiveRecord::Base; end
 
   setup do
     @connection = ActiveRecord::Base.connection
     @connection.create_table("postgresql_geometrics") do |t|
-      t.column :a_line_segment, :lseg
-      t.column :a_box, :box
-      t.column :a_path, :path
-      t.column :a_polygon, :polygon
-      t.column :a_circle, :circle
+      t.line    :a_line
```

Comment:
Does this `line` type not follow the same restrictions for PostgreSQL version? I think we should check the dump in `PostgreSQLGeometricLineTest` as well and remove that from here.